### PR TITLE
DOC: Update installation instructions

### PR DESCRIPTION
**Pull request by jhlegarreta**
Update installation instructions:
- Update the dependencies versions.
- `Canopy` is no longer maintained/available, so remove it.
- Update the instal command to use `pip` as the build system has been changed and there is no `setup.py` file.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Draft until tag/release.

File: doc/installation_instructions.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Downloading the source code from git::
 Installing::
 
   cd tract_querier
-  python setup.py install
+  pip install
```
Comment (demianw) [inline doc/installation_instructions.rst:28]: this should be `pip install .`

**Review by demianw — CHANGES_REQUESTED** [review]

File: doc/installation_instructions.rst (hunk 1 above)
Comment (jhlegarreta) [reply doc/installation_instructions.rst]: Thanks for checking. Fixed in f991c57.

**Comment by tashrifbillah** [comment]
@demianw , I was going to submit a PR but I see that somebody addressed this before. Did you ever deploy this merge? I am asking because the installation instruction is still outdated:

**Comment by tashrifbillah** [comment]
@demianw , I was going to submit a PR but I see that somebody addressed this before. Did you ever deploy this merge? I am asking because the installation instruction is still outdated:
http://demianw.github.io/tract_querier/installation_instructions.html#installation

**Pull request merged**